post_install.sh

```diff
@@ -2,10 +2,22 @@
 
 
 export ENDPOINT=http://localhost:9082/api/v3
+wallet=/root/keystore/godWallet.json
```

Comment:
godWallet.json in in gochain- btp/data. Use relative path for it, instead of absolute path